### PR TITLE
updates to Truss Guide

### PR DESCRIPTION
**Pull request by amy-meng**
outlined edits on Slack here: https://trussworks.slack.com/archives/C02STUN4R54/p1643296349042200

edits in this PR are indicated with a green check

### CODE REVIEW

File: docs/trussels.md [hunk 1]
```diff
@@ -362,29 +351,15 @@ They are all great and approachable.
...
-## Resource: Departments and Practice Areas
-Truss has three departments led by the founders:
-
-* BizOps (business operations), led by Jen Leech (COO)
-* Key Practices (the product/service we sell), led by Mark Ferlatte (CTO)
-* Sales (commercial and government), led by Everett Harper (CEO)
+* **Amy Meng** Associate Director of Delivery Management
+* **Carla Silvey** Director of Product Management
+* **Carmen Bocanegra** Director of Research and Design 
+* **Elizabeth Eady** (Temporary) Director of Infrastructure Engineering
```
Comment (ferlatte) [inline docs/trussels.md:357]: Please correct Eady's title.

File: docs/trussels.md [hunk 2]
```diff
@@ -362,29 +351,15 @@ They are all great and approachable.
...
+* **Hannah Kane** VP of Delivery (oversees Client Success Managers and Delivery Managers)
```
Comment (ferlatte) [inline docs/trussels.md:358]: Please correct Hannah's title. In general, Paylocity is the source of truth here.

**Review by ferlatte — CHANGES_REQUESTED** [review]
Ugh, GitHub. Please resolve the titles, then I think this is ok to merge.

File: docs/trussels.md (hunk 1 above)
Comment (amy-meng) [reply docs/trussels.md:357]: what is the correct title?

File: docs/trussels.md (hunk 2 above)
Comment (amy-meng) [reply docs/trussels.md:358]: Paylocity lists Hannah's title as "Vice President, Delivery" -- should it be something different?

File: docs/trussels.md (hunk 1 above)
Comment (amy-meng) [reply docs/trussels.md:357]: Paylocity lists Eady as "Senior Manager, Infrastructure Engineering". is that right?

Comment (ferlatte) [reply docs/trussels.md:357]: Yes that sounds right. Whatever Paylocity says is what is "true"; if it seems wrong, then we need to work with People & Culture to fix it there first.

File: docs/trussels.md (hunk 2 above)
Comment (ferlatte) [reply docs/trussels.md:358]: LOL. OK, hoisted on my own Paylocity petard. Apologies for assuming.

**Comment by ferlatte** [comment]
Now that this has been broken apart and has conflicts, it might make sense to close this out and resubmit whatever changes are remaining? Up to you @amy-meng

**Pull request closed without merging**